spec/features/studies/accession_all_samples_spec.rb

```diff
@@ -3,12 +3,16 @@
 require 'rails_helper'
 
 describe 'Accession all samples', :accessioning_enabled do
+  include AccessionV1ClientHelper
+
   let!(:user) { create(:user, api_key: configatron.accession_local_key) }
   let!(:study) { create(:open_study, accession_number: 'ENA123', samples: create_list(:sample_for_accessioning, 5)) }
 
   before do
     Delayed::Worker.delay_jobs = false
-    allow(Accession::Request).to receive(:post).and_return(build(:successful_sample_accession_response))
+    allow(Accession::Submission).to receive(:client).and_return(
```

Comment:
Minor: 
maybe 'Accession::EBISubmission' as 'Submission' is already a big thing in SS?
('Request' was also a big thing mind. And it is namespaced.)